component/updates.go

```diff
@@ -8,34 +8,36 @@ import (
 )
 
 // UpdatesContent returns the entire content of updates tab.
-func UpdatesContent(rps []*model.RepoPresentation, checkingUpdates bool) []vecty.MarkupOrChild {
-	return []vecty.MarkupOrChild{
+func UpdatesContent(rps []*model.RepoPresentation, checkingUpdates bool) vecty.List {
+	return vecty.List{
 		&Header{},
 		elem.Div(
 			vecty.Markup(prop.Class("center-max-width")),
 			elem.Div(
-				updatesContent(rps, checkingUpdates)...,
+				vecty.Markup(prop.Class("content")),
+				&updatesHeader{RPs: rps, CheckingUpdates: checkingUpdates},
+				updatesContent(rps),
 			),
 		),
 	}
 }
 
-func updatesContent(rps []*model.RepoPresentation, checkingUpdates bool) []vecty.MarkupOrChild {
-	var content = []vecty.MarkupOrChild{
-		vecty.Markup(prop.Class("content")),
-	}
-
-	content = append(content,
-		updatesHeader{
-			RPs:             rps,
-			CheckingUpdates: checkingUpdates,
-		}.Render()...,
-	)
-
+func updatesContent(rps []*model.RepoPresentation) vecty.List {
+	var content vecty.List
 	wroteInstalledUpdates := false
 	for _, rp := range rps {
 		if rp.UpdateState == model.Updated && !wroteInstalledUpdates {
-			content = append(content, InstalledUpdates())
+			content = append(content,
+				elem.Heading3(
+					vecty.Markup(
+						// This element is mixed with keyed siblings, we choose an
+						// arbitrary key here that can not conflict with any siblings.
+						vecty.Key("__gps_installed_updates"),
+						vecty.Style("text-align", "center"),
```

Comment:
Is `vecty.Style("text-align", "center")` really needed? Why? It wasn't there before.